packages/app/src/app/overmind/factories.ts

```diff
@@ -61,3 +61,26 @@ export function withLoadApp<T>(
     }
   };
 }
+
+export function withOwnedSandbox<T>(
+  continueAction: AsyncAction<T>
+): AsyncAction<T> {
+  return async (context, payload) => {
```

Comment:
If you make this an arrow function, this can be an implicit return.

```ts
export const withOwnedSandbox = async <T>(
  continueAction: AsyncAction<T>
): AsyncAction<T> => async (context, payload) => {
  // ...
};
```

Reply:
Jup, personally I think it reduces readability a bit, but we have a linting rule on this... fixed :)